src/node_modules/app/components/AccessibilityMenu.svelte

```diff
@@ -20,4 +68,32 @@
 		left: 50%;
 		margin-left: -320px;
 	}
+	header {
+		grid-area: indicators;
+		display: grid;
+		grid-template-areas: "upper" "lower";
+		grid-template-rows: 50% 50%;
+	}
+	menu {
+		grid-area: proppanel;
+	}
+	nav {
+		grid-area: nav;
+		display: grid;
+		grid-template-areas: "upper" "lower";
+		grid-template-rows: 50% 50%;
+	}
+	label {
+		display: block;
+		text-align: center;
+	}
+	button, out {
+		width: 39px;
+		height: 39px;
+		box-sizing: border-box;
+		text-align: center;
+		border: thin solid;
```

Comment:
These buttons have a double border, they should have a single border as per the design.